.vscode/settings.json

```diff
@@ -0,0 +1,48 @@
+{
+  "cSpell.words": [
+    "asax",
+    "cijo",
```

Comment:
😆 this is interesting.. I think the only place where people's name are relevant are the main readme which lists maintainer/approver/etc. 
Could we exclude a section from spell check? Or anything to avoid listing all names in dictionary..

Reply:
This is like how I mentioned I just sort of... _ignored_ the fact that `weirdlongwordinaurl` isn't really a word. The spell checker see's it's not a word, it reports it's not a word, but I just didn't do anything about it. We could always just ignore that the spell checker reports people's names. However, I thought it would be more pleasant for the folks listed to see that the checker "knows their name" rather than reporting it as a misspelled word. I don't think _literally every contributor_ shows up in here so the list shouldn't change that often, right?

You can set up an `ignorePaths` configuration for the spell checker to ignore certain full documents, but it seemed like it was valuable to actually spell check the main README since it's... the main README. There is not a way to say "ignore just this one section" that I'm aware of. Perhaps the listing of all the contributors, etc., should be on a separate page that's linked from the main README; that'd allow for exclusion of a whole document.

Should the [GitHub pipeline action that checks spelling](https://github.com/open-telemetry/opentelemetry-dotnet/blob/7ed97be90841221ec2451d8d14a622768cb214b5/.github/workflows/sanitycheck.yml#L17) get fixed and start working, I don't think that has the ability to ignore sections of docs, either.

Reply:
Want to hear what other people think of having people's name in dictionary.
@open-telemetry/dotnet-approvers

Reply:
It seems weird to me but what @tillig explained makes sense. I'm fine either way.

My only question - is this guarded by CI? If not, it's likely that things will start to rot over time. (my personal take is that if something is not controlled by CI, then it's likely to be unsuccessful in the long run)

Reply:
*this*, as of now, is not CI guarded. Once https://github.com/open-telemetry/opentelemetry-dotnet/issues/3031 is fixed, this would be guarded by CI, as adding a new maintainer/approver etc. without modifying the dictionary will break CI. Though there can be extra names in the dictionary, if approvers/maintainers leave a lot.

I'll approve this change now. Once https://github.com/open-telemetry/opentelemetry-dotnet/issues/3031 is addressed, we'll no longer have to worry about rotting.